test/fork.ts

```diff
@@ -0,0 +1,89 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {expect} from 'chai';
+import {describe, it, before} from 'mocha';
+import {logger, octokit, setup} from './util';
+import * as sinon from 'sinon';
+import {fork} from '../src/github-handler/fork-handler';
+
+before(() => {
+  setup();
+});
+
+describe('Fork', async () => {
+  const upstreamOwner = 'upstream-owner';
+  const upstreamRepo = 'upstream-repo';
+  const responseData = await import('./fixtures/create-fork-response.json');
+
+  describe('Octokit createFork function', () => {
+    it('Is called with the correct values', async () => {
+      const createRefResponse = {
+        headers: {},
+        status: 200,
+        url: 'http://fake-url.com',
+        data: responseData,
+      };
+      // setup
+      const stub = sinon
```

Comment:
Are we using nock purely to disable net connect? I wonder if we could potentially get away with just using `sinon` for the time being.

Reply:
I asked for that precisely for that reason. Do you have another suggestion?

Reply:
Yes. Nock is solely used as a guarantee that we never connect to the internet. Sinon stubs definitely could do the trick, but isn't fool-proof.

@chingor13 thoughts?